cmd/speedtest-spinner.go

```diff
@@ -269,6 +270,13 @@ func (m *speedTestUI) View() string {
 					} else {
 						dataItem = append(dataItem, whiteStyle.Render(humanize.IBytes(nodeResult.TX/uint64(nodeResult.TXTotalDuration.Seconds())))+"/s")
 					}
+					// show TTFB
+					if uint64(nodeResult.RX) == 0 {
+						dataError += "- RXTotalDuration are zero"
+						dataItem = append(dataItem, crossTickCell)
+					} else {
+						dataItem = append(dataItem, whiteStyle.Render(fmt.Sprintf("%.2f ns", float64(nodeResult.TXTotalSpentDuration.Nanoseconds())/float64(nodeResult.RX))))
```

Comment:
You should perhaps use some other time of humanize value here instead of converting it into ns. 

My suggestion is stick to `nodeResult.TXTotalSpentDuration/nodeResult.RX`